Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Spectrometer Changelog
 
+## v2.12.2
```

Comment:
We have `v2.12.1` release but it's not in change log. 

https://github.com/fossas/spectrometer/releases/tag/v2.12.1

Reply:
Do you mind adding the notes from 2.12.1 into the changelog?